DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs

```diff
@@ -98,7 +98,8 @@ ELSE 0
                             'Supervisor') AS SignOffRoleName,
                         SA.SignOffRequestorStatement,
                         SA.ManageSupervisorsDescription,
-                        CA.NonReportable
+                        CA.NonReportable,
+					 DA.FirstName_deprecated +' '+ DA.LastName_deprecated AS DelegateName

```

Comment:
Hi Sherif. As the column name suggests, it is deprecated. We now store names in the User table. Please make use of the User table to fetch the names. Also, the join would change accordingly.

Reply:
ok

Reply:
Done